### ISSUE TITLE
Support [Explicit] for NUnit 

### ISSUE THREAD

**Issue by 304NotModified** [axodox/AxoCover#134]
Please support the  `[Explicit]` attribute in NUnit

> The Explicit attribute causes a test or test fixture to be ignored unless it is explicitly selected for running. 

([source](http://nunit.com/index.php?p=explicit&r=2.2.2)

** expected **

test methods or test classes with the `[Explicit]` attribute are skipped by default (inconclusive), when doing "run tests" on an higher level

** current result **

the explicit tests are running when  "run tests" on an higher level

**Comment by hoontje** [comment]
I think this would be a valuable enhancement

**Comment by LucasKatsuo** [comment]
Yep, I'm waiting for this feature by one year, would be very valuable have this. =)
